src/Makefile.am

```diff
@@ -1,5 +1,5 @@
 AM_CFLAGS = -std=gnu11 $(GLIB_CFLAGS) -Wall -I$(top_srcdir)/ps2emu-kmod
-AM_LDFLAGS = $(GLIB_LIBS) $(GLIB_LDFLAGS)
```

Comment:
the original was already a bug anyway. `AM_LDFLAGS` are the linker **flags**, semantically similar to the `CFLAGS`. `GLIB_LIBS` should go into `LDADD` (see the [docs](https://www.gnu.org/software/automake/manual/html_node/Linking.html)) or `progname_LDADD`. So this should really be:
```
AM_LDFLAGS = $(GLIB_LDFLAGS)
LDADD = $(GLIB_LIBS)
```